mdx-models/src/main/java/com/mx/path/model/mdx/accessor/feature/ErrorDescriptor.java

```diff
@@ -0,0 +1,27 @@
+package com.mx.path.model.mdx.accessor.feature;
+
+/**
+ * ErrorDescriptor representing different error descriptors in the application.
+ * <p>
+ * This enum provides a centralized way to manage error descriptors used
+ * throughout the application. Each constant represents a specific error
+ * condition that may arise during application operations.
+ * </p>
+ * <p>
+ * Usage:
+ * <p>
+ * Use these constants to represent error conditions when throwing
+ * exceptions or logging errors, ensuring consistency across the application.
+ * </p>
+ */
+public enum ErrorDescriptor {
+  INSUFFICIENT_FUNDS,
+  ACCOUNT_NOT_FOUND,
+  UNKNOWN_ERROR;
+  // todo: All the expected ErrorDescriptors will be added here.
+
+  @Override
+  public String toString() {
+    return name();
+  }
+}
```

Comment:
Sorry. I probably should have caught this earier. I think it would be better to make an enumeration of these per feature. That will make usage much clearer and make it easier to explain each error descriptor.